src/utils/fields.js

```diff
@@ -84,7 +85,7 @@ const FOLDER_FIELDS_TO_FETCH = [
 ];
 
 // Fields needed for the sidebar
-const SIDEBAR_FIELDS_TO_FETCH = [
+const SIDEBAR_FIELDS_TO_FETCH: Array<string> = [
```

Comment:
I had to add type because flow was showing error 
<img width="898" alt="image" src="https://github.com/user-attachments/assets/55d237fb-7f98-4047-a3da-24d9e2ae2f8c">

I could also add same type to other field arrays in this file to be consistent or move concat to functions in which we fetch

Reply:
Hm. I'm not seeing this error locally. Flow is pretty terrible, but it should be able to infer one string array from another. Are you using the local Flow binary installed with the project? If so, maybe try removing the type annotations to see if it was a temporary issue?

Reply:
Removed type annotation from `SIDEBAR_FIELDS_TO_FETCH`, but left one on `SIDEBAR_FIELDS_TO_FETCH_ARCHIVE`, without this one the flow error was still showing